lib/blockscore/base.rb

```diff
@@ -53,22 +52,22 @@ def self.resource
       @resource ||= Util.to_underscore(to_s.split('::').last)
     end
 
-    def self.api_url
-      'https://api.blockscore.com/'
-    end
-
     def self.endpoint
       fail NotImplementedError, ABSTRACT_WARNING if equal?(Base)
 
-      "#{api_url}#{Util.to_plural(resource)}"
+      API_URL + Util.to_plural(resource)
     end
 
     def persisted?
-      !id.nil? && !attributes[:deleted]
+      !id.nil? && !attributes.fetch(:deleted, false)
```

Comment:
Yeah, AST complexity is a heuristic we use *when all other things are equal* to determine a choice between multiple equivalent pieces of code. However if one is easier to understand than the other, then it's not equal, it's superior to the other choices, and should be ranked higher.